src/features/main/container/MainContainer.js

```diff
@@ -50,15 +55,15 @@ const MainContainer = ({ navigation }) => {
       <Main
         onExitApp={exitApp}
         handleShowScoreModal={handleShowScoreModal}
-        handleSelectGameNumberAndShowDescriptionModal={
-          handleSelectGameNumberAndShowDescriptionModal
+        handleSetGameNameAndShowDescriptionModal={
+          handleSetGameNameAndShowDescriptionModal
```

Comment:
prop명이랑 handler 함수명 다르게 변경만 하면 될 것 같습니다~!
저는 보통 아래와 같이 사용한다라고 알고 있는데 여기서는 2개의 일을 하고 있어서 우선 handle만 on으로 변경해도 되지 않을까 싶습니다
prop 명 => on+subject+event
handler 함수명 => handle+subject+event

```suggestion
        onSetGameNameAndShowDescriptionModal={
          handleSetGameNameAndShowDescriptionModal
```